### DISCUSSION TITLE
Integrating Semantic Kernel with Azure Function App

### DISCUSSION THREAD

**Discussion by rahulsinghania1206** [microsoft/semantic-kernel#7579] [Q&A] [answered]
I am attempting to connect to an Azure Function App using the Semantic Kernel from my local system. I plan to connect via the Azure Function App’s API.
Could someone guide me on how to trigger this using the Semantic Kernel? Additionally, how should I set up the code and the Function App? Is any authorization required, and if so, how can it be configured?
I’ve reviewed the documentation but haven’t had any success. If anyone has experience with this, your help would be greatly appreciated. Sample code with an end-to-end flow would be especially helpful.

**Comment by rajyraman** [comment, marked as answer]
@rahulsinghania1206 - I was reading this post today https://techcommunity.microsoft.com/t5/azure-integration-services-blog/integrate-logic-app-workflows-as-plugins-with-semantic-kernel/ba-p/4210854 after totally missing the [initial announcement](https://devblogs.microsoft.com/semantic-kernel/connect-logic-apps-1400-connectors-to-semantic-kernel/#use-logic-apps-with-semantic-kernel-as-a-plugin). Since Logic Apps is built on Function runtime as well, you should be able to follow most of what is in this post to hookup Function app to SK.
